docs/readmes/basics/prerequisites.md

```diff
@@ -43,6 +43,7 @@ Development can occur from multiple OS's, where **macOS** and **Ubuntu** are **e
    pyenv global 3.8.10
    pip3 install ansible fabric3 jsonpickle requests PyYAML
    vagrant plugin install vagrant-vbguest
+   vagrant plugin install vagrant-disksize
```

Comment:
Nit: Would be nicer to put these two vagrant commands on one line like you do in the CI

Reply:
done